Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM php:7.2
+
+# Install composer and required packages
+RUN apt-get update \
```

Comment:
Here possible shrink image size by inserting `--no-install-recommends` and `rm -rf /var/lib/apt/lists/*`

Reply:
done